src/libertem/io/utils.py

```diff
@@ -6,7 +6,11 @@
     import pwd
 
     def get_owner_name(full_path, stat):
-        return pwd.getpwuid(stat.st_uid).pw_name
+        try:
+            name = pwd.getpwuid(stat.st_uid).pw_name
+        except KeyError:
+            name = stat.st_uid
```

Comment:
So this is hit if the file owner UID has no name, correct? Surprised we didn't hit this before :)

Reply:
Yes, precisely! It tends to happen within Docker containers...